org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/views/locations/IssueLocationsView.java

```diff
@@ -114,7 +114,11 @@ public boolean equals(Object obj) {
 
     public FlowRootNode(MarkerFlow flow) {
       this.flow = flow;
-      children = flow.getLocations().stream().map(FlowNode::new).collect(toList());
+      children = flow.getLocations().stream()
+        // SLE-388 - "Highlight-only" locations don't have a message
```

Comment:
Just for my information, is it a common practice to leave such comments in the code ?